config/manifest.yaml

```diff
@@ -3,65 +3,64 @@ dist:
   name: otelcol-custom
   module: github.com/liatrio/liatrio-otel-collector
   description: Basic OTel Collector distribution for Developers
-  otelcol_version: 0.112.0
```

Comment:
This was removed in https://github.com/open-telemetry/opentelemetry-collector/issues/11405

Backwards compatibility was removed and now the version defaults to the version of the builder that compiles the collector.